config/plugins/interactive_environments/rstudio/config/rstudio.ini.sample

```diff
@@ -0,0 +1,11 @@
+[main]
+# This cannot be changed
+password_auth = True
+# Other
+apache_urls = False
+ssl = False
+
+[docker]
+command = docker
+image = erasche/docker-rstudio-notebook:dev
+
```

Comment:
@erasche I don't think this tag exists (https://registry.hub.docker.com/u/erasche/docker-rstudio-notebook/tags/manage/) - it seems to for IPython (https://registry.hub.docker.com/u/bgruening/docker-ipython-notebook/tags/manage/). Can you mimic @bgruening's IPython setup or switch this to ``:latest``?